.github/workflows/run-acceptance-tests.yml

```diff
@@ -379,17 +416,17 @@ jobs:
       run: |-
         pip3 install virtualenv==20.0.23
         pip3 install pipenv
-    - name: Run docker-compose
-      run: docker-compose -f testing/docker-compose.yml up --build -d
-    - name: Run setup script
-      run: testing/setup.sh
     - name: Install dependencies
       run: make install_${{ matrix.language}}_sdk
     - name: Install gotestfmt
       uses: GoTestTools/gotestfmt-action@v2
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         version: v2.5.0
+    - name: Ensure upstream
+      run: make upstream
+    - name: Setup local Keycloak
+      run: make -C upstream local
```

Comment:
I don't see a `local` make target - where is that coming from?

Reply:
The `-C` flag tells make to execute in that directory, so `make -C upstream local` is largely equivalent to `cd upstream && make local; cd -`. The `local` target is coming from upstream's makefile.